src/main/java/com/jcabi/xml/ClasspathInput.java

```diff
@@ -121,10 +121,12 @@ public String getEncoding() {
     @Override
     public String getStringData() {
         try {
-            return IOUtils.toString(
+            final String data = IOUtils.toString(
                 this.stream,
                 Charset.forName("UTF-8")
             );
+            this.stream.close();
```

Comment:
@amihaiemil I have doubts about closing the stream right here.
Usually, it's client's responsibility to close the stream if the stream was provided by him.